ERCS/erc-7272.md

```diff
@@ -0,0 +1,221 @@
+---
+eip: 7272
+title: Ethereum Access Token
+description: A protocol for authorizing function calls from an off-chain service
+author: Chris Chung (@0xpApaSmURf), Raphael Roullet (@ra-phael)
+discussions-to: https://ethereum-magicians.org/t/eip-7272-ethereum-access-token/14945
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-07-03
+requires: 712
+---
+
+## Abstract
+
+An Ethereum Access Token (EAT) is an [EIP-712](./eip-712.md) conformant, signed message, used by off-chain services to grant Ethereum accounts access to specific on-chain resources. EATs share similarities with JWTs; both are used for short-lived authorizations. However Ethereum Access Tokens are specifically designed to be verified on-chain and tailored to authorize smart contract function calls.
+
+## Motivation
+
+While other proposals tackle authentication ([ERC-4361](./eip-4361.md)) or authorization in a more narrow way ([ERC-2612](./eip-2612.md)), this specification allows developers to add a layer of access control to any function they create with minimal changes. It is best suited for use cases where end users should only be able to access specific on-chain resources themselves directly, by way of sending a transaction, provided they have been granted authorization by an off-chain service first. Examples of such scenarios include an off-chain verifier assessing eligibility requirements (e.g by verifying verifiable credentials) to mint a token or to interact with a smart contract that requires a certain compliance status.
```

Comment:
Just a question on this. Since our EIP doesn't strictly depend on other EIPs, but is referenced for literature, would that still qualify as a dependency? If so, would we be able to link to other EIPs in say an appendix where it does not lie in the main body of the proposal in order to avoid such dependency cascade?

Reply:
No, any link between proposals creates a dependency.

For example, the authors of ERC-4361 could completely rewrite it and change the topic, and links here would no longer make sense.